frontend/website/docs/my-first-transaction.md

```diff
@@ -67,6 +67,9 @@ While you're waiting take a look at your daemon logs for new blocks being genera
 
 Finally we get to the good stuff, sending our first transaction! For testing purposes, there's already an [echo service](https://github.com/CodaProtocol/coda-automation/tree/master/services/echo-service) set up that will immediately refund your payment minus the transaction fees.
 
+!!! warning
+    If two requests go to the echo service within the same 5min window, the echo service may eat your funds. You'll still get testnet points even if this happens, but you'll lose the tokens you've sent.
```

Comment:
```suggestion
    Currently, there is a known issue with the echo service that prevents it from sending multiple transactions within the same 5 minute window. As such, it may not return funds currently, but you will still receive Testnet Points for completing the challenge.
```